crates/events/src/correlation_id.rs

```diff
@@ -9,10 +9,12 @@ use std::{
     sync::atomic::{AtomicUsize, Ordering},
 };
 
+use serde::{Deserialize, Serialize};
```

Comment:
are you expecting `serde` also on `trbfv` as well? there we were following the `protobuf` style which was a default of the lib.

Reply:
This is a good point - it is necessary as all actor messages need to be serializable. I will go over this and work out a good way to integrate protobuf

Reply:
yes, thanks. I can try to remove protobufs and use serde where needed on fhe.rs